packages/hash/frontend/src/blocks/page/createPlaceholderPlugin/createPlaceholderPlugin.tsx

```diff
@@ -19,18 +24,30 @@ export const createPlaceholderPlugin = (renderPortal: RenderPortal) => {
     key: placeholderPluginKey,
     state: {
       init() {
-        return { focused: false };
+        return defaultState;
       },
       apply(tr, state) {
-        const focused = tr.getMeta(placeholderPluginKey);
+        const partialState = tr.getMeta(placeholderPluginKey);
 
-        // return the old state if transaction does not have focused meta
-        if (focused === undefined) {
-          return state;
+        return partialState ? { ...state, ...partialState } : state;
+      },
+    },
+    view(view) {
+      const update = () => {
+        const { editable } =
+          placeholderPluginKey.getState(view.state) ?? defaultState;
+
+        if (view.editable !== editable) {
+          view.dispatch(
+            view.state.tr.setMeta(placeholderPluginKey, {
+              editable: view.editable,
+            }),
+          );
         }
+      };
 
-        return { focused };
-      },
+      update();
+      return {};
```

Comment:
As discussed on Slack, this needs to return `update` and `EditorConnection` needs updating to avoid crashes when meta-transactions (i.e, not changes to the document) are dispatched before we have the page from collab https://github.com/hashintel/hash/pull/995

Reply:
Fixed here fyi https://github.com/hashintel/hash/pull/953/commits/1e4489bd598d46443be9487011c30b195a5e4692